llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -243,8 +244,8 @@
 #include "llvm/Transforms/Utils/CanonicalizeAliases.h"
 #include "llvm/Transforms/Utils/CanonicalizeFreezeInLoops.h"
 #include "llvm/Transforms/Utils/CountVisits.h"
-#include "llvm/Transforms/Utils/Debugify.h"
 #include "llvm/Transforms/Utils/DXILUpgrade.h"
+#include "llvm/Transforms/Utils/Debugify.h"
```

Comment:
Weird. clang-format requirement.

Reply:
Right, 
```
#include "llvm/Transforms/Utils/DXILUpgrade.h"
```
line was added in llorg, so clang-format check was avoided. I included `SYCLPropagateJointMatrixUsage.h` above and that's why clang-format triggered on this. This incorrect placing of the header will bring us one more merge conflict during the internal pulldown.